react/components/EXPERIMENTAL_TableTree/README.md

```diff
@@ -32,9 +34,19 @@ function ToolbarExample() {
   const [inputValue, setInputValue] = React.useState('')
   const [displayItems, setDisplayItems] = React.useState(items)
 
+  const onToggle = ({ checkedItems }) => {
+    console.log(checkedItems)
```

Comment:
:thinking:

Reply:
It is just to represent an action that could be done onToggle. (i found `alert` annoying)

Reply:
Fine for a readme indeed.